StatePrinter.Tests/IntegrationTests/DictionaryTest.cs

```diff
@@ -803,13 +836,13 @@ public void Dictionary_complexKeySimpleValue()
                 var expected = @"<Root type='Dictionary(Person, Int32)'>
     <Element type='KeyValuePair(Person, Int32)'>
         <key type='Person'>
-            <Name>""Douglas""</Name>
+            <Name>Douglas</Name>
```

Comment:
I recall wanting the " in there to specify that we are dealing with a string - but I totally agree with you it has no place in neither json nor xml - perhaps not even in the "CurlyBraceStyle" - it was supposed to be a "best readable" format for programmers.. I think your change is an improvement